components/brave_new_tab_ui/containers/newTab/settings/solidColorBackgroundOption.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import {
+  StyledCustomBackgroundOption,
+  StyledCustomBackgroundOptionSolidColor,
+  StyledSelectionBorder
+} from '../../../components/default'
+
+interface Props {
+  color: string
+  selected: boolean
+  useSolidColorBackground: (color: string) => void
+}
+
+class SolidColorBackgroundOption extends React.PureComponent<Props, {}> {
```

Comment:
As this is a new component, it might be worth creating it as a functional component. (optional, seeing as you've already made it :smile: )

In general, I think we want to convert everything to functional components , but I think it's going to take us a while :laughing: